pkg/web/buildConfigsHandler.go

```diff
@@ -198,23 +198,23 @@ func newBuildConfigObject(data BuildConfigCreateRequest, namespace string) (*mob
 func (mbch *MobileBuildConfigsHandler) Create(c echo.Context) error {
 	reqData := new(BuildConfigCreateRequest)
 	if err := c.Bind(reqData); err != nil {
-		return err
+		return c.String(http.StatusBadRequest, err.Error())
```

Comment:
can we log the errors as well, or is it already logged?

Reply:
in the log there is just the http response code without the error message. Is it enough or we should also log the error message?

Reply:
I think we need to error message.